main/main.cpp

```diff
@@ -1170,7 +1170,9 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 #ifdef TOOLS_ENABLED
 		if (!editor && !project_manager) {
 #endif
-			OS::get_singleton()->print("Error: Can't run project: no main scene defined.\n");
+			const String error_msg = "Error: Can't run project: no main scene defined in the project.\n";
+			OS::get_singleton()->print("%s", error_msg.ascii().get_data());
```

Comment:
```suggestion
			OS::get_singleton()->print("%s", error_msg.utf8().get_data());
```

Reply:
Done; I also changed the other occurrence of `.ascii()` in `main.cpp`.